src/tools/perf/rte/mad/perftest_mad.c

```diff
@@ -715,3 +703,18 @@ ucs_status_t cleanup_mad_rte(struct perftest_context *ctx)
 
     return UCS_OK;
 }
+
+UCS_STATIC_INIT {
+    static struct perftest_rte plugin_mad_rte = {
+        .setup   = setup_mad_rte,
+        .cleanup = cleanup_mad_rte
+    };
+
+    mad_rte.report = rte_report_common_func;
```

Comment:
why not defining it during initalization?

Reply:
`rte_report_common_func` is a function pointer declared in `ucx_perftest`. The function needs some code linked in `ucx_perftest` and is not a static initializer so it cannot be used in mad_rte global variable above, I did not find a cleaner way tbh.